lib/users/create.js

```diff
@@ -6,10 +6,6 @@ module.exports = function(req, res) {
     if(!(req.body instanceof Array)) req.body = [req.body];
 
     async.map(req.body, async.reflect((body, callback) => createUser(req.pgdb, body.userId, body.name, callback)), (error, users) => {
-        if(error) {
-            req.log.error({ error }, "Error while creating user");
-            return res.sendStatus(500);
-        }
```

Comment:
`reflect` is confusing, I agree.

Having a map where only part of the functions are actually executed is even more confusing. :)